web/src/css/media_queries.scss

```diff
@@ -0,0 +1,52 @@
+/*
+Add media queries/breakpoints related rules here.
+*/
+@media screen and (max-width: 640px) {
+    .d-md-block {
+      display: block;
+    }
+    .d-md-none {
+      display: none;
+    }
+  
+    .subtitle {
+      flex-direction: column;
+      text-align: center;
+  
+      h5 {
+        line-height: 1.5rem;
+      }
+    }
+  
+    .mdc-data-table {
+      .mdc-data-table__header-cell,
+      .mdc-data-table__cell {
+        max-width: 100px;
+      }
+      .mdc-data-table__cell {
+        white-space: break-spaces;
+        line-height: 1.4;
+  
+        &.result {
+          text-align: right;
+        }
+      }
+      .embedded_region {
+        font-style: italic;
+        margin-top: 10px;
```

Comment:
As a rule, let's avoid px rules unless we really need them. This looks better to me as `0.4rem`.

Reply:
Before:

![image](https://user-images.githubusercontent.com/1175430/151604726-121b22cd-80ac-426d-aa53-e19040a99610.png)

After:

![image](https://user-images.githubusercontent.com/1175430/151604809-c4b36b45-ebb5-45fe-bc60-452af8629ebd.png)